src/autocomplete_abstract.coffee

```diff
@@ -195,7 +197,9 @@ class AutocompleteAbstract
     handleFilterBlur: ->
 
     createEmptyList: ->
-        $list = $ '<div></div>'
+        notice = if @options.showNoResultsNotice then 'No Results Found…' else ''
+
+        $list = $ '<div>#{notice}</div>'
```

Comment:
It's actually not okay to have it like this. While autocomplete is waiting for data from an external data source you will see 'No Results Found…' which is not true. Is it not enough to just handle it in `handleEmptyList `?

You should also use double quotes to substring `notice`.